doc/tutorials/dockerfile.rst

```diff
@@ -82,12 +82,13 @@ Preparing your Dockerfile
 For a Dockerfile to work on Binder, it must meet the following requirements:
 
 1. It must install a recent version of Jupyter Notebook.
+
    This should be installed via ``pip`` with the `notebook` package.
    So in your dockerfile, you should have a command such as:
 
    .. code-block:: Dockerfile
 
-       RUN pip install --no-cache-dir notebook==5.*
+      RUN pip install --no-cache-dir notebook
```

Comment:
I'm changing a four year old line of code. We have support for `notebook==6.*` now!